### DISCUSSION TITLE
试一下可以发吗

### DISCUSSION THREAD

**Discussion by mr-xiyu** [share121/inter-knot#4768] [其他]
看到可以回复一下吗，第一次用，不知道能不能用呢

**Comment by qingshuiglaze** [comment]
回复一下，这个没法看回复还是比较难受的，也没法看自己发的东西，不过邮箱至少有提示

**Reply by mr-xiyu** [reply]
感谢回复，不过我发现一段时间以后是可以在原帖看到自己的回复的，不过这个时间有点长

**Reply by qingshuiglaze** [reply]
不，你刷新一下就行了，邮件上倒是有直接跳转到帖子的地方
